sonar-java-plugin/src/main/resources/org/sonar/l10n/java/rules/java/S6889.html

```diff
@@ -9,11 +9,16 @@ <h2>Why is this an issue?</h2>
     <ul>
```

Comment:
Actually, I think this list is not complete - `VirtualDisplay` is missing, for instance